usecases/Concrete/knowledgeGraph/emodul/emodul_validation.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
+from time import sleep
+from turtle import pos
+import requests
+import os
+import sys
+from pathlib import Path
+from git import Repo
+
+baseDir0 = Path(__file__).resolve().parents[0]
+baseDir1 = Path(__file__).resolve().parents[1]
+baseDir2 = Path(__file__).resolve().parents[2]
+ontologyPath = os.path.join(baseDir2,'ConcreteOntology')
+metadataPath = os.path.join(baseDir0,'E-modul-processed-data/emodul_metadata.csv')
+graphPath = os.path.join(baseDir0,'E-modul-processed-data/EM_Graph.ttl')
+#importedOntologiesPath = os.path.join(baseDir2,'GraphCreation/Ontologies')
+processedDataPath = os.path.join(baseDir0,'E-modul-processed-data')
+
+# clone and run rdfconverter docker
+Repo.clone_from('https://github.com/Mat-O-Lab/RDFConverter.git', os.path.join(baseDir1, 'RDFConverter'))
+subprocess.run(['docker-compose', '-f', os.path.join(baseDir1, 'RDFConverter/docker-compose.yml'), 'up', '-d', 'app'], stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
+
+passed = [] #list of filenames of files that passed validation
+failed = [] #list of tuples (filenames, reports) of files that failed validaton
+
+# read shacl shape
+with open(os.path.join(baseDir1, 'shape.ttl'), 'r') as f:
+    shacl_shape = f.read()
+
+sleep(3)
+s = requests.session()
```

Comment:
IMO it would be great to have some comments on this interface (what the the docker container starts a server that is accessed via https ...)